### DISCUSSION TITLE
What it takes to support the web/browser besides ios/android

### DISCUSSION THREAD

**Discussion by benc43** [amugofjava/anytime_podcast_player#114] [Q&A]
Hi
Understand the current version does not support flutter web (tried compile with Flutter(Chrome) and doesn't work). My mvp project, custom podcast with a special player, needs to start off without the need to install the app (testing idea and users acceptance)
Any help appreciated

**Comment by amugofjava** [comment]
Hi @benc43,

I'm not sure I understand your question. Are you trying to create an embedded web player for your own Podcast app?

I don't have a lot of experience with Flutter web as I've focused on mobile and a bit of desktop, but I'll give you some pointers if I can.

**Reply by benc43** [reply]
hi thank you for the response. I basically want to run Anytime Podcast Player using a browser. so users do not need to install the app on their mobile phone.
if in visual studio code debug with Flutter (Chrome) configuration does not generate any error and when the browser is activated main.dart.js is loaded without error but a blank white screen appears. clicking on the blank browser screen has nothing happen but when reloaded the follow errors appears in Debug Console

ext.flutter.connectedVmServiceUri: (-32603) Unexpected DWDS error for callServiceExtension: WipError -32000 Execution context was destroyed.
Failed to set vm service URI: ext.flutter.connectedVmServiceUri: (-32603) Unexpected DWDS error for callServiceExtension: WipError -32000 Execution context was destroyed.. Deep links to DevTools will not show in Flutter errors.
Failed to set DevTools server address: ext.flutter.activeDevToolsServerAddress: (-32603) Unexpected DWDS error for callServiceExtension: WipError -32000 Execution context was destroyed.. Deep links to DevTools will not show in Flutter errors.

looks like codes pertaining to any mobileSettings etc is incompatible in the web environment

any thoughts?

thanks

**Comment by amugofjava** [comment]
Ah! you are trying to get Anytime to work in the browser. Anytime targets mobile and makes a lot of use of the dart:io package which cannot be used with Flutter web. You would need to update/replace the code that makes use of this package, or try a package such as [universal_io](https://pub.dev/packages/universal_io/install) which is a drop in replacement for dart:io that also handles web.

There are other areas of the code that you would need to check and adapt to running on the web.

**Reply by benc43** [reply]
Hi Ben
Many thanks. I figure that much. thanks for the tip on universal_io. when I manage to port to web I will share here
